composer.json

```diff
@@ -15,6 +15,12 @@
     },
     "require": {
         "php": ">=5.4",
+        "ext-pdo_mysql": "*",
+        "ext-mcrypt": "*",
+        "ext-intl": "*",
+        "ext-gd": "*",
```

Comment:
or a script using the `pre-install-cmd` hook https://getcomposer.org/doc/articles/scripts.md